languages/ruby/lib/ruby-bitwarden-sdk.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'dry-types'
+
+require_relative 'schemas'
+require_relative 'extended_schemas/schemas'
+require_relative 'command_runner'
+require_relative 'bitwarden_lib'
+require_relative 'projects'
+require_relative 'secrets'
+
+class BitwardenClient
+  attr_reader :bitwarden, :project_client, :secrets_client
+
+  def initialize
+    api_url = ENV['BITWARDEN_API_URL'] || 'https://api.bitwarden.com'
+    device_type = ENV['BITWARDEN_DEVICE_TYPE'] || 'MacOsDesktop'
```

Comment:
We don't want the users to be able to override device type, it should always be SDK. I.e. use null.